### ISSUE TITLE
Bugfix: Inconsistent RUN_ID values when using instances

### ISSUE THREAD

**Issue by georgemccabe** [dtcenter/METplus#2596]
## Describe the Problem ##

If a use case utilizes instance identifiers (see [Instance Names in Process List](https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#instance-names-in-process-list)), then different RUN_ID values are logged.

### Expected Behavior ###
The same run ID should be used in all logs for a given METplus run.

### Environment ###
Describe your runtime environment: Any

### To Reproduce ###
Describe the steps to reproduce the behavior:
*1. Copy METplus/parm/use_cases/met_tool_wrapper/Example/Example.conf to another directory*
*2. Modify the config to change the process list and add an instance header*:

Change:

```
PROCESS_LIST = Example
```

to:

```
PROCESS_LIST = Example, Example(my_instance)

[my_instance]

[config]
```

*3. Run use case and review ID values in the logs:*

> 05/21 15:45:31.591 metplus.**7d86825e** (config_util.py:131) INFO: ****************************************
> 05/21 15:45:31.592Z metplus.**c3256cbe** (runtime_freq_wrapper.py:351) INFO: Processing forecast lead 3 hours

### Relevant Deadlines ###
*List relevant project deadlines here or state NONE.*

### Funding Source ###
*Define the source of funding and account keys here or state NONE.*

### Assignee ###
- [ ] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [ ] Review default **alert** labels
- [ ] Select **component(s)**
- [ ] Select **priority**
- [ ] Select **requestor(s)**

### Milestone and Projects ###
- [ ] Select **Milestone** as the next bugfix version
- [ ] Select **Coordinated METplus-X.Y Support** project for support of the current coordinated release
- [ ] Select **METplus-Wrappers-X.Y.Z Development** project for development toward the next official release

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [ ] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdataio](https://github.com/dtcenter/METdataio/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Bugfix Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **main_\<Version>**.
Branch name: `bugfix_<Issue Number>_main_<Version>_<Description>`
- [ ] Fix the bug and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/appendixA.html#metplus-components-python-packages) table.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **main_\<Version>**.
Pull request: `bugfix <Issue Number> main_<Version> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issue
Select: **Milestone** as the next bugfix version
Select: Coordinated METplus-X.Y Support project for support of the current coordinated release
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Complete the steps above to fix the bug on the **develop** branch.
Branch name:  `bugfix_<Issue Number>_develop_<Description>`
Pull request: `bugfix <Issue Number> develop <Description>`
Select: **Reviewer(s)** and **Development** issue
Select: **Milestone** as the next official version
Select: **METplus-Wrappers-X.Y.Z Development** project for development toward the next official release

- [ ] Close this issue.

**Comment by georgemccabe** [comment]
This fix will be included with the changes for #2612. This issue can be closed when that issue is resolved.

**Comment by georgemccabe** [comment]
This was fixed with PR #2698
